ui/com/msg-view/card.jsx

```diff
@@ -214,13 +227,24 @@ export default class Card extends React.Component {
     const unreadReplies = countReplies(msg, m => !m.isRead)
     const isViewingRaw = this.state.isViewingRaw
     const channel = msg && msg.value && msg.value.content && msg.value.content.channel
-
+    
     const dropdownOpts = [
-      { value: 'copy-link',  label: <span><i className="fa fa-external-link" /> Copy ID</span> },
-      { value: 'toggle-raw', label: <span><i className={isViewingRaw?'fa fa-envelope-o':'fa fa-gears'} /> View {isViewingRaw?'Msg':'Data'}</span> },
-      (isDownvoted) ?
-        { value: 'unflag',   label: <span><i className="fa fa-times" /> Unflag</span> } :
-        { value: 'flag',     label: <span><i className="fa fa-flag" /> Flag</span> }
+      {
+        value: 'copy-link',
+        Com: props => <ClipboardBtn component='li' data-clipboard-text={msg.key} onSuccess={props.onClick}>
+          <i className="fa fa-external-link" /> Copy ID
+        </ClipboardBtn>
```

Comment:
@pfraze this is pretty interesting. When this `item.Com` is used later, it doesn't get called like `item.Com(this.props)` 
Is that something that is implicitly done when jsx is transpiled  ?

i.e.
```jsx
<item.Com key={i} onClick={onClick} />

// becomes something like: 
<item.Com(this.props) key={i} onClick={onClick} />
```

tried googling stuff like _dynamic jsx component function_ ... not that much luck. If you've got keywords that would improve my search those would be great.